codalab/client/local_bundle_client.py

```diff
@@ -168,9 +168,15 @@ def get_bundle_target(self, target):
 
     def get_worksheet_uuid(self, base_worksheet_uuid, worksheet_spec):
         if worksheet_spec == '' or worksheet_spec == worksheet_util.HOME_WORKSHEET:
-            # Default worksheet name: take the username.
             worksheet_spec = spec_util.home_worksheet(self._current_user_name())
             return self.new_worksheet(worksheet_spec, True)
+        elif spec_util.is_dashboard(worksheet_spec):
+            home_worksheet_spec = spec_util.home_worksheet(self._current_user_name())
```

Comment:
Why create home worksheet when people ask for dashboard?  I don't think should have it.